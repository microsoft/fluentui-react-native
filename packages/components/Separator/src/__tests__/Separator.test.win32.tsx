import * as React from 'react';
import { Separator } from '..';
import * as renderer from 'react-test-renderer';

it('Separator default', () => {
  const tree = renderer.create(<Separator />).toJSON();
  expect(tree).toMatchSnapshot();
});

it('Separator all props & tokens', () => {
  const CustomSeparator = Separator.customize({ separatorWidth: 15, color: 'red' });
  const tree = renderer.create(<CustomSeparator vertical />).toJSON();
  expect(tree).toMatchSnapshot();
});
