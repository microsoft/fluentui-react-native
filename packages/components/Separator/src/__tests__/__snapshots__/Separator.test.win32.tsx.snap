// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Separator all props & tokens 1`] = `
<View
  style={
    Object {
      "borderColor": "red",
      "borderLeftWidth": 15,
    }
  }
/>
`;

exports[`Separator default 1`] = `
<View
  style={
    Object {
      "borderTopWidth": 1,
    }
  }
/>
`;
